kpops/cli/main.py

```diff
@@ -227,7 +265,7 @@ def schema(
             gen_config_schema()
 
 
-@app.command(  # pyright: ignore[reportGeneralTypeIssues] https://github.com/rec/dtyper/issues/8
+@app.command(  # type: ignore[reportGeneralTypeIssues] https://github.com/rec/dtyper/issues/8
```

Comment:
why did you change these?

Reply:
Forgot to undo them, initially thought Pyright didn't recognize `# pyright: ignore` since it didn't seem to work and the `type` comments work, but it turns out that `# type: ignore[...]` ignores all errors no matter the brackets' content

Newer versions of Pyright moved some errors into new categories, so I _correctly_ get diagnostics despite the ignore comments

Reply:
fixed

Reply:
Ah yeah, I noticed that too. Think we should update our bundled Pyright